python-sdk/example_dags/example_google_bigquery_gcs_load_and_save.py

```diff
@@ -22,19 +22,29 @@
 from astro.files import File
 from astro.table import Metadata, Table
 
+default_args = {
+    "owner": "airflow",
+    "retries": 1,
+    "retry_delay": 0,
+}
+
 with DAG(
     dag_id="example_google_bigquery_gcs_load_and_save",
     schedule_interval=None,
     start_date=timezone.datetime(2022, 1, 1),
     catchup=False,
+    is_paused_upon_creation=False,
+    default_args=default_args,
 ) as dag:
     # [START load_file_http_example]
     t1 = aql.load_file(
         task_id="load_from_github_to_bq",
         input_file=File(
             path="https://raw.githubusercontent.com/astronomer/astro-sdk/main/tests/data/imdb_v2.csv"
         ),
-        output_table=Table(name="imdb_movies", conn_id="bigquery", metadata=Metadata(schema="astro")),
+        output_table=Table(
+            name="imdb_movies", conn_id="google_cloud_platform", metadata=Metadata(schema="astro")
```

Comment:
Let's not hard-code the connection ID here, but define it on the top of the file:
```
ASTRO_GCP_CONN_ID = os.getenv("ASTRO_GCP_CONN_ID", "google_cloud_default")
```

The [same comment](https://github.com/astronomer/astro-sdk/pull/1937/files#r1198689257) about the connection ID applies here.